src/topology/bornology/hom.lean

```diff
@@ -0,0 +1,119 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import topology.bornology.basic
+
+/-!
+# Locally bounded maps
+
+This file defines locally bounded maps between bornologies.
+
+We use the `fun_like` design, so each type of morphisms has a companion typeclass which is meant to
+be satisfied by itself and all stricter types.
+
+## Types of morphisms
+
+* `locally_bounded_map`: Locally bounded maps. Maps which preserve boundedness.
+
+## Typeclasses
+
+* `locally_bounded_map_class`
+-/
+
+open bornology filter function
+
+variables {F α β γ δ : Type*}
+
+/-- The type of bounded maps from `α` to `β`, the maps which send a bounded set to a bounded set. -/
+structure locally_bounded_map (α β : Type*) [bornology α] [bornology β] :=
+(to_fun : α → β)
+(comap_cobounded_le' : (cobounded β).comap to_fun ≤ cobounded α)
+
+/-- `locally_bounded_map_class F α β` states that `F` is a type of bounded maps.
+
+You should extend this class when you extend `locally_bounded_map`. -/
+class locally_bounded_map_class (F : Type*) (α β : out_param $ Type*) [bornology α]
+  [bornology β]
+  extends fun_like F α (λ _, β) :=
+(comap_cobounded_le (f : F) : (cobounded β).comap f ≤ cobounded α)
+
+export locally_bounded_map_class (comap_cobounded_le)
+
+/-
+TODO: Add instances from `continuous_linear_map_class` (warning, does not exist yet) to
+`locally_bounded_map_class` here, or alternately in the file with `continuous_linear_map_class`.
+-/
```

Comment:
```suggestion
```
As mentioned in zulip, I don't see that this will ever be possible in this file.